platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/builder/PlatformBuilder.java

```diff
@@ -147,15 +153,20 @@ public final class PlatformBuilder {
      * @param selfId              the ID of this node
      * @param softwareVersion     the software version of the application
      * @param initialState        the genesis state supplied by the application
+     * @param consensusEventStreamName the consensusEventStreamName
+     * @param rosterHistory       the RosterHistory
```

Comment:
These javadocs are not very helpful. I suggest "the name of the directory where the consensus event stream is written" and "the roster history provided by the application to use at startup"